english/chess/CheesePuzzle.md

```diff
@@ -0,0 +1 @@
+[it sort of makes me want to cry...](http://2.bp.blogspot.com/-8VIW5EHOXis/VC6PctbH9kI/AAAAAAAAM0s/n_5JtDdrhQY/s1600/IMG_3434.JPG)
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.